crypto/s2n_hmac.c

```diff
@@ -200,6 +200,7 @@ S2N_RESULT s2n_hmac_state_validate(struct s2n_hmac_state *state)
 
 int s2n_hmac_init(struct s2n_hmac_state *state, s2n_hmac_algorithm alg, const void *key, uint32_t klen)
 {
+    notnull_check(state);
```

Comment:
How come you do not need this assumption in the harness?

Reply:
This is not an assertion. If `state` is NULL, then the function will return an error code; therefore, I don't need an assumption, this is a valid computational path.